cluster/manifests/02-admission-control/teapot.yaml

```diff
@@ -519,8 +519,6 @@ webhooks:
     matchConditions:
       - name: 'exclude-privileged-groups'
         expression: 'request.userInfo.groups.all(g, !(g in ["okta:common/administrator", "zalando:administrator"]))'
-      - name: 'exclude-postgres-admins'
-        expression: 'request.userInfo.groups.all(g, !(g in ["okta:common/postgres-admin"]))'
```

Comment:
This is now handled in-code in admission-controller version `master-229`: https://github.bus.zalan.do/teapot/admission-controller/pull/266

The main reason was the additional allow-rule for `postgres-operator` which just made the rules more complicated. The exclusion rules for `okta:common/administrator`, `system:masters` etc. and are mainly used to resolve chicken-and-egg problems. We don't have those for postgres exec handling and therefore we can handle them inside admission-controller's code.